src/main/resources/assets/js/components/AppListComponent.jsx

```diff
@@ -99,6 +100,11 @@ var AppListComponent = React.createClass({
               </span>
             </th>
             <th className="text-right">
+              <span className={headerClassSet}>
```

Comment:
Let's add some sorting to this, so it is possible to sort the applications based on the health (status).